packages/block-library/src/latest-posts/edit.js

```diff
@@ -417,9 +439,10 @@ export default withSelect( ( select, props ) => {
 	const { getEntityRecords, getMedia } = select( 'core' );
 	const { getSettings } = select( 'core/block-editor' );
 	const { imageSizes, imageDimensions } = getSettings();
+	const catIds = categories.map( ( cat ) => cat.id );
```

Comment:
You assume there are already some categories set, which for newly added blocks this is not the case. This is why the meta box tests fail [here](https://travis-ci.com/github/WordPress/gutenberg/jobs/296849373#L1018), I believe.

Therefore try to check for `categories` and if empty set `catIds` to an empty array so that `map` works later on:

```JS
const catIds =
               categories && categories.length > 0
                       ? categories.map( ( cat ) => cat.id )
                       : [];